src/python/pants/engine/internals/native_engine.pyi

```diff
@@ -155,23 +155,23 @@ class PySnapshot:
 
 class PyExecutionRequest:
     def __init__(
-        self, poll: bool, poll_delay_in_ms: int | None, timeout_in_ms: int | None
+        self, *, poll: bool, poll_delay_in_ms: int | None, timeout_in_ms: int | None
```

Comment:
These being explicit kwargs is slightly misleading, because they aren't kwargs on the receiving side: they must be passed in-order/positionally.

Reply:
I don't think that's true for class methods, but will double check. Fwit, that was original motivation for this PR when I was using `/` to enforce pos args, until finding it's not possible to use with Py37.